README.rst

```diff
@@ -10,8 +10,8 @@ Scrapy
    :target: https://pypi.python.org/pypi/Scrapy
    :alt: Supported Python Versions
 
-.. image:: https://img.shields.io/travis/scrapy/scrapy/master.svg
-   :target: https://travis-ci.org/scrapy/scrapy
+.. image:: https://github.com/scrapy/scrapy/workflows/Build/badge.svg
```

Comment:
I'm not sure this name exists now, and it seems the names are different for Windows and Ubuntu now, sso there should be 2 badges.

Reply:
Updated, thanks for letting me know. Related to this: should I add jobs for macOS?

Reply:
It'd be nice, but did we have them before this PR? 
I think that's fine to do it separately, unless you're "in a flow" now.

Reply:
Done!